changelog/lld_mingw.dd

```diff
@@ -0,0 +1,7 @@
+Windows: Visual C++ and the Windows SDK are no longer required to build 64-bit executables
+
+The Windows installer now adds platform libraries built from the MinGW definitions and
+a wrapper library for the VC2010 shared C runtime. When building COFF object files with
+-m64 or -m32mscoff and no Visual Studio installation is detected or
```

Comment:
nit: tick marks around -m64 and -m32mscoff.

Reply:
Done.